### PR TITLE
downloader/common.py: Order all topologies for print_all

### PR DESCRIPTION
**Pull request by epyeoh**
Current print_all printing all topologies in non-sorted manner.
Order all topologies for print_all to print sorted topologies.

Signed-off-by: Yeoh Ee Peng <ee.peng.yeoh@intel.com>

### CODE REVIEW

File: tools/downloader/common.py [hunk 1]
```diff
@@ -321,8 +321,11 @@ def load_topologies(config):
 # requires the --print_all, --all, --name and --list arguments to be in `args`
 def load_topologies_from_args(parser, args):
     if args.print_all:
+        print_list = []
```
Comment (IRDonch) [inline tools/downloader/common.py]: I would prefer a more Pythonic way to populate this, e.g. `print_list = [top.name for top in load_topologies(args.config)]`.

Comment (epyeoh) [reply tools/downloader/common.py]: @IRDonch thanks for reviewing and sharing on Pythonic way. I had made the suggested changes and rebase.

**Comment by IRDonch** [comment]
Thanks, code looks good. Could you also edit the README to reflect the changed output from `--print_all`?

**Comment by epyeoh** [comment]
@IRDonch yes, let me look at the README.

**Comment by epyeoh** [comment]
@IRDonch I had updated the README with guide for --print_all and the latest output.

File: tools/downloader/README.md [hunk 2]
```diff
@@ -68,6 +68,108 @@ The cache format is intended to remain compatible in future Open Model Zoo
...
+ssd_mobilenet_v2_coco
+text-detection-0002
+text-recognition-0012
+vehicle-attributes-recognition-barrier-0039
+vehicle-detection-adas-0002
+vehicle-detection-adas-binary-0001
+vehicle-license-plate-detection-barrier-0106
+vgg16
+vgg19
+```
```
Comment (IRDonch) [inline tools/downloader/README.md]: This isn't quite what I meant. There already is a description of `--print_all` down below; I just want you to update the example output in that description.

Comment (epyeoh) [reply tools/downloader/README.md]: @IRDonch I had made the update for the example output for existing ./TOOL.py --print_all. Is that what your looking for?

Comment (IRDonch) [reply tools/downloader/README.md]: Almost, but I don't think the _full_ output needs to be listed. It unnecessarily bloats the document and creates a maintenance burden. Please abbreviate it the way it's abbreviated now.

Comment (epyeoh) [reply tools/downloader/README.md]: Updated and it was abbreviated as before.

**Comment by IRDonch** [comment]
Jenkins: ok to test

**Pull request merged**